static/styles/portico/portico-signin.scss

```diff
@@ -922,6 +922,14 @@ button.login-social-button {
 
     #enter-realm-button {
         margin-top: 14px;
+        background-color: hsl(0, 0%, 100%);
+        box-shadow: 0px 1px 3px hsla(0, 0%, 0%, 0.3), 0px 0px 5px hsla(0, 0%, 0%, 0.1);
+        color: hsl(0, 0%, 34%);
+    }
+
+    #enter-realm-button:hover {
+        background-color: hsl(0, 0%, 98%);
```

Comment:
color of button was completely fine before but this change makes it a little bit lighter to be visible on first go